jointwt/twtxt
# Add polling for media task

##### Summary
We want to poll for the task that we queued (video uploads this time)
##### Component Name
area/frontend
##### Test Plan
- Run it locally and try uploading videos
- Make sure that the loading state of the spinner is working properly
##### Additional Information
How it looks like when it errors
![onerror](https://user-images.githubusercontent.com/15314237/95006551-01afeb80-05c3-11eb-9f24-d500a5981f4f.gif)
Note: i've sent to dm how it looks like on success
